<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeviceWindow</class>
 <widget class="QMainWindow" name="DeviceWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1015</width>
    <height>664</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <widget class="QDockWidget" name="dockInfo">
      <property name="minimumSize">
       <size>
        <width>250</width>
        <height>138</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>260</width>
        <height>150</height>
       </size>
      </property>
      <property name="features">
       <set>QDockWidget::NoDockWidgetFeatures</set>
      </property>
      <property name="windowTitle">
       <string>디바이스 정보</string>
      </property>
      <widget class="QWidget" name="dockWidgetContents">
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>장치명</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="editName"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>버전</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="editVersion"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>IP 주소</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="editIP"/>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>MCU온도</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="editMCUTemp"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="0" column="1" rowspan="4">
     <widget class="QFrame" name="frame">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>99999</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_7">
       <item row="0" column="0">
        <widget class="QCustomPlot" name="plotGraph" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>400</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QFrame" name="frame_2">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>450</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout_6">
          <item row="0" column="0">
           <layout class="QGridLayout" name="gridLayout">
            <item row="2" column="0">
             <widget class="FormPort" name="widget_9" native="true"/>
            </item>
            <item row="2" column="3">
             <widget class="FormPort" name="widget_12" native="true"/>
            </item>
            <item row="1" column="1">
             <widget class="FormPort" name="widget_6" native="true"/>
            </item>
            <item row="1" column="2">
             <widget class="FormPort" name="widget_7" native="true"/>
            </item>
            <item row="1" column="0">
             <widget class="FormPort" name="widget_5" native="true"/>
            </item>
            <item row="0" column="0">
             <widget class="FormPort" name="widget_1" native="true"/>
            </item>
            <item row="1" column="3">
             <widget class="FormPort" name="widget_8" native="true"/>
            </item>
            <item row="2" column="1">
             <widget class="FormPort" name="widget_10" native="true"/>
            </item>
            <item row="0" column="2">
             <widget class="FormPort" name="widget_3" native="true"/>
            </item>
            <item row="0" column="1">
             <widget class="FormPort" name="widget_2" native="true"/>
            </item>
            <item row="0" column="3">
             <widget class="FormPort" name="widget_4" native="true"/>
            </item>
            <item row="2" column="2">
             <widget class="FormPort" name="widget_11" native="true"/>
            </item>
            <item row="3" column="0">
             <widget class="FormPort" name="widget_13" native="true"/>
            </item>
            <item row="3" column="1">
             <widget class="FormPort" name="widget_14" native="true"/>
            </item>
            <item row="3" column="2">
             <widget class="FormPort" name="widget_15" native="true"/>
            </item>
            <item row="3" column="3">
             <widget class="FormPort" name="widget_16" native="true"/>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QDockWidget" name="dockSetting">
      <property name="minimumSize">
       <size>
        <width>253</width>
        <height>150</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>260</width>
        <height>150</height>
       </size>
      </property>
      <property name="features">
       <set>QDockWidget::NoDockWidgetFeatures</set>
      </property>
      <property name="windowTitle">
       <string>디바이스 설정</string>
      </property>
      <widget class="QWidget" name="dockWidgetContents_2">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>스캔 주기 (sec)</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSpinBox" name="sbScanCycle">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="value">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="btnSetScanCycle">
          <property name="text">
           <string>SET</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>검출 주기 (sec)</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="sbDetectionPeriod">
          <property name="maximum">
           <number>255</number>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QPushButton" name="btnSetDetectionPeriod">
          <property name="text">
           <string>SET</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>절체 횟수 (회)</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QSpinBox" name="sbSwitchingTime">
          <property name="maximum">
           <number>255</number>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QPushButton" name="btnSetSwitchingTime">
          <property name="text">
           <string>SET</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>기준파워 (dBm)</string>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QPushButton" name="btnSetThreshold">
          <property name="text">
           <string>SET</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QDoubleSpinBox" name="dsbPdThreshold">
          <property name="minimum">
           <double>-99.000000000000000</double>
          </property>
          <property name="maximum">
           <double>20.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QDockWidget" name="dockSetting_2">
      <property name="minimumSize">
       <size>
        <width>250</width>
        <height>217</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>260</width>
        <height>217</height>
       </size>
      </property>
      <property name="features">
       <set>QDockWidget::NoDockWidgetFeatures</set>
      </property>
      <property name="windowTitle">
       <string>전체널 제어</string>
      </property>
      <widget class="QWidget" name="dockWidgetContents_3">
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QGroupBox" name="groupBox_4">
          <property name="title">
           <string>모드</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_8">
           <item row="0" column="0">
            <widget class="QPushButton" name="btnAutoAll">
             <property name="text">
              <string>전채널 자동</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QPushButton" name="btnManualAll">
             <property name="text">
              <string>전채널 수동</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QGroupBox" name="groupBox_6">
          <property name="title">
           <string>경로</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_9">
           <item row="0" column="0">
            <widget class="QPushButton" name="btnMainAll">
             <property name="text">
              <string>전채널 메인</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QPushButton" name="btnSubAll">
             <property name="text">
              <string>전채널 서브</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="3" column="0">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1015</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>파일</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>설정</string>
    </property>
    <addaction name="actionSetDeviceInfo"/>
    <addaction name="actionSetPortName"/>
    <addaction name="actionSetAlarmMask"/>
    <addaction name="actionPdTable"/>
    <addaction name="actionGetLog"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSetPortName">
   <property name="text">
    <string>포트 이름 설정</string>
   </property>
  </action>
  <action name="actionSetAlarmMask">
   <property name="text">
    <string>알람 마스크 설정</string>
   </property>
  </action>
  <action name="actionSetDeviceInfo">
   <property name="text">
    <string>디바이스 정보 설정</string>
   </property>
  </action>
  <action name="actionPdTable">
   <property name="text">
    <string>보정데이터 설정</string>
   </property>
  </action>
  <action name="actionGetLog">
   <property name="text">
    <string>로그 데이터</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FormPort</class>
   <extends>QWidget</extends>
   <header>formport.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
